seedlet-blocks/sass/blocks/_navigation.scss

```diff
@@ -0,0 +1,19 @@
+.wp-block-navigation__responsive-container {
+	&.has-modal-open {
+		.wp-block-navigation-link {
+			//NOTE: For reasons I cannot explain... if I set this to use the --wp--preset--font-family--headings
+			//these values (which are what are set) are not respected in the modal.
+			font-family: var(--font-headings, 'Playfair Display', Georgia, Times, serif);
```

Comment:
This is strange and I don't understand why this is.  Perhaps somebody could help me figure out why using the variable in this space wasn't working...
```
font-family: var(--wp--preset--font-family--headings);
```

Reply:
Odd bug - I think it's safe to add `font-family: var(--wp--preset--font-family--headings);`

It works fine for me: 

![W9CsKOiNW1](https://user-images.githubusercontent.com/905781/119419520-bc2f6280-bcfa-11eb-8c83-6874c799240c.gif)